src/vivarium_public_health/population/base_population.py

```diff
@@ -202,6 +202,31 @@ def _validate_config_for_deprecated_keys(self) -> None:
                 f"Public Health. Use the new key '{mapper[key]}' instead."
             )
 
+    def _load_population_structure(self, builder: Builder) -> pd.DataFrame:
+        return load_population_structure(builder)
+
+
+class ScaledPopulation(BasePopulation):
+    """Component for scaling the population structure based on a scaling factor."""
```

Comment:
Can you expand on this docstring. What is this component for? How do you use it?